duckduckgo.pot

```diff
@@ -14730,12 +14834,27 @@ msgctxt "made_by"
 msgid "by %s"
 msgstr ""
 
+# smartling.placeholder_format=C
+#. An abreviated unit of time indicating an age, e.g. "2d"
+msgid "d"
+msgstr ""
+
 # smartling.placeholder_format=C
 #. An abbreviated form of "day" that can be displayed in a small space e.g. "1d"
 msgctxt "published date for videos"
 msgid "d"
 msgstr ""
 
+# smartling.placeholder_format=C
+#. Describing a unit of time in the context of a period ago, e.g. "1 *day* ago"
+msgid "day"
+msgstr ""
```

Comment:
@LoveAndCoding Are these strings used for "1 day ago" and stuff like that? Have we considered using [`Intl.RelativeTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat) for this stuff? It's supported in all major browsers and you can easily wrap it in a function to supply a fallback to English when not supported. That's a much better way to handle these. If for whatever reason we don't want to use that API (but please explain), we should still include the whole string, with the numbers otherwise translators will have a hard time getting these right.

cc @franfaccin

Reply:
Honestly, no clue. These were not from my change. The ones I added are all tagged with `noresults` ctx.

That said, while it is widely supported now (yay!) it doesn't meet our minimum browser requirements as I understand them (e.g. FF > 36 iirc). So we may not be able to use this yet unless/until we bump that baseline. But I am 100% in favor of using that because it makes so many things so much better. The `Intl` namespace does.

Reply:
Cool. Not your changes, so let's not block this. I've been advocating for using `Intl` with English fallback for years. Seems like I haven't done a good job at that. 🙂